replay_parser/src/packet_parser/events/entity_method/update_arena.rs

```diff
@@ -189,6 +191,37 @@ pub struct VehicleKilled {
     victim_id:    i32,
     killer_id:    i32,
     equipment_id: i32,
-    reason:       i32,
-    //TODO: More fields in later versions
+    attack_reason: AttackReason,
+
+    #[version([1, 17, 0, 0])]
+    num_vehicles_affected: Option<i32>,
```

Comment:
```suggestion
    pub attack_reason: AttackReason,

    #[version([1, 17, 0, 0])]
    pub num_vehicles_affected: Option<i32>,
```
Because of #6, I made the other fields on this struct public already